### PR TITLE
Fixed several smoke tests

### PR DESCRIPTION
**Pull request by olenagerasimova**
Fixed `smoke-tests.yml` github workflow, go and debian smoke tests.

### CODE REVIEW

File: .github/workflows/smoke-tests.yml [hunk 1]
```diff
@@ -13,7 +13,7 @@ jobs:
       - uses: actions/checkout@v2
       - uses: actions/setup-java@v1
         with:
-          java-version: 11
+          java-version: 17
       - uses: actions/cache@v1
         with:
           path: ~/.m2/repository
```
Comment (genryxy) [inline .github/workflows/smoke-tests.yml:19]: @olenagerasimova is it okay that here in key is used `jdk-11-maven` instead of `jdk-17`?

Comment (olenagerasimova) [reply .github/workflows/smoke-tests.yml:19]: @genryxy thanks, corrected

File: .github/workflows/smoke-tests.yml [hunk 2]
```diff
@@ -13,11 +13,11 @@ jobs:
...
-          key: ${{ runner.os }}-jdk-11-maven-${{ hashFiles('**/pom.xml') }}
+          key: ${{ runner.os }}-jdk-17-maven-${{ hashFiles('**/pom.xml') }}
           restore-keys: ${{ runner.os }}-jdk-11-maven-
       - run: mvn -B package -Pdocker-build
```
[the remark this answers is not in the document]
Comment (olenagerasimova) [reply .github/workflows/smoke-tests.yml]: @genryxy fixed, thanks again

**Review by genryxy — APPROVED** [review]
@olenagerasimova thanks, looks good to me
@g4s8 could you please check / merge

**Pull request merged**